chia/consensus/block_header_validation.py

```diff
@@ -834,8 +835,7 @@ def validate_finished_header_block(
     blocks: BlockRecordsProtocol,
     header_block: HeaderBlock,
     check_filter: bool,
-    expected_difficulty: uint64,
-    expected_sub_slot_iters: uint64,
+    vs: ValidationState,
     check_sub_epoch_summary: bool = True,
     prev_ses_block: Optional[BlockRecord] = None,
```

Comment:
this function should no longer take `prev_ses_block`. This is part of `ValidationState`

Reply:
I have that one in a separate branch as I don't want to alter it in this one.

Reply:
That slipped back in due to a `Tuple` vs `tuple` conflict.